docs/integration/event-driven/index.md

```diff
@@ -1,7 +1,53 @@
-# Event Driven
+# Principles for Event-Driven Architecture
 
-!!! warning "Work In Progress"
-    The whole Event Driven section is work in progress
+We strongly believe that an Event-Driven Architecture is the best approach to solve many of the problems which Bane NOR is facing. With our initiative to become more service oriented with service teams, this will be increasingly important as we need to architect and work with events, and not systems. Information about events must be shared for everyone to consume.The following is a list of principles we want to enforce in Bane NOR to establish such an Event-Driven Architecture. They shall support our overall company goals like cost reduction, reduction of time-to-market, but also ensure a high level of reliability.
```

Comment:
```suggestion
(...) They **must** support our overall company goals like cost reduction, reduction of time-to-market, but also ensure a high level of reliability.
```
Should phrase in accordance with the RFC standard used in the Integration Guidelines.